ts/src/program/namespace/transaction.ts

```diff
@@ -16,8 +16,13 @@ export default class TransactionFactory {
     const txFn: TransactionFn<IDL, I> = (...args): Transaction => {
       const [, ctx] = splitArgsAndCtx(idlIx, [...args]);
       const tx = new Transaction();
+      if(ctx.preInstructions && ctx.preInstructions.length && ctx.instructions && ctx.instructions.length){
```

Comment:
```suggestion
      if (ctx.preInstructions && ctx.instructions) {
```

Is the `.length` needed here?